### PR TITLE
Fix links in APIs (mostly)

### PR DESCRIPTION
**Pull request by arvindsv**
* Change http to https
* Also, add /current/ where needed. Redirect won't work properly with that fragment otherwise.

### CODE REVIEW

**Comment by arvindsv** [comment]
If there's no problem with this and it gets merged, I assume I'll need to change it in all the documentation as well. Will do if it's merged.

File: server/webapp/WEB-INF/rails.new/app/presenters/api_v1/config/pipeline_config_representer.rb [hunk 1]
```diff
@@ -31,7 +31,7 @@ class PipelineConfigRepresenter < ApiV1::BaseRepresenter
       end
 
       link :doc do |opts|
-        'http://api.go.cd/#pipeline-config'
+        'https://api.go.cd/current/#pipeline_config'
```
Comment (varshavaradarajan) [inline server/webapp/WEB-INF/rails.new/app/presenters/api_v1/config/pipeline_config_representer.rb]: @arvindsv, in the current api docs, it is pipeline-config. Did you change the api docs as well?

**Comment by ketan** [comment]
Would it not be easier to setup redirects at the index.html page over at api.go.cd?

File: server/webapp/WEB-INF/rails.new/app/presenters/api_v1/config/pipeline_config_representer.rb (hunk 1 above)
Comment (arvindsv) [inline server/webapp/WEB-INF/rails.new/app/presenters/api_v1/config/pipeline_config_representer.rb]: No. This is just a search-replace in the codebase.

**Comment by arvindsv** [comment]
Maybe it is easier, but I feel that correcting this is ... more correct.

File: server/webapp/WEB-INF/rails.new/app/presenters/api_v1/config/pipeline_config_representer.rb (hunk 1 above)
Comment (arvindsv) [inline server/webapp/WEB-INF/rails.new/app/presenters/api_v1/config/pipeline_config_representer.rb]: But, I see what you mean. Maybe I need to rebase and re-do it. I will.

Comment (varshavaradarajan) [inline server/webapp/WEB-INF/rails.new/app/presenters/api_v1/config/pipeline_config_representer.rb]: Yeah, There was one PR today morning to replace pipeline_config with pipeline-config. :)

Comment (arvindsv) [inline server/webapp/WEB-INF/rails.new/app/presenters/api_v1/config/pipeline_config_representer.rb]: I remember. The one where @ketan kept changing his mind. :)

I've now corrected this.

**Comment by arvindsv** [comment]
@ketan thinks we should just redirect in the docs. I think otherwise. The next person who has an opinion gets to break the tie. I won't begrudge you if the PR gets closed because of your decision, as long as there is a decision. :)

**Comment by ketan** [comment]
Can you make this 2 PRs? One for URL change in representers and another for docs and header/footer?

On another note, u was considering replacing the footer on all pages to be the same as/similar to the one in the SPA. Thoughts?

**Comment by arvindsv** [comment]
> Can you make this 2 PRs? One for URL change in representers and another for docs and header/footer?

@ketan: Ok, closing it.

> On another note, u was considering replacing the footer on all pages to be the same as/similar to the one in the SPA. Thoughts?

As long as it is consistent and looks good, I'm fine with it. @naveenbhaskar might be able to help, to say if it looks and feels ok. It shouldn't feel like a franken-app where going from an SPA page to a non-SPA page is a jarring experience.

**Pull request closed without merging**